### ISSUE TITLE
Space invitations via API

### ISSUE THREAD

**Issue by coatless** [rstudio/rscloud#13]
Presently, space invitations must be initiated on RStudio Cloud's web interface. When trying to onboard a class, it would be great if the API allowed for the programmatic creation of the invitation. Moreover, if the student's account isn't already created, then the student could be prompted to make an account.

https://github.com/rstudio/rscloud/blob/85b286847dbe30db19b09a4dde5dd38b1aa4b6f3/R/invitations.R#L54-L72

**Comment by coatless** [comment]
Actually, this looks like the use case is covered in `space_member_add()`. Perhaps the wording here could be changed and/or `@seealso [space_member_add()]` could be added.

**Comment by tareefk** [comment]
Yes, exactly . I will add it in the next block of time I spend on fixing/improving the package.

**Comment by gullahoo** [comment]
Just curious what the status of this issue is, in light of #16 it appears a student I am aware of is not being granted any space, let alone prompted for space to be 'allocated' or rather, granted.

Just a suggestion, perhaps, but unions in your underlying code's SEXP types may not be best for granting additional space!
